aci/data_source_aci_psuinstpol.go

```diff
@@ -0,0 +1,49 @@
+package aci
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ciscoecosystem/aci-go-client/v2/client"
+	"github.com/ciscoecosystem/aci-go-client/v2/models"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceAciPowerSupplyRedundancyPolicy() *schema.Resource {
+	return &schema.Resource{
+		ReadContext:   dataSourceAciPowerSupplyRedundancyPolicyRead,
+		SchemaVersion: 1,
+		Schema: AppendBaseAttrSchema(AppendNameAliasAttrSchema(map[string]*schema.Schema{
+			"admin_rdn_m": {
```

Comment:
```suggestion
			"administrative_state": {
```

Reply:
Yes. We generally have been trying more recently to match closer to the UI than to the model (API)